client/config/config.go

```diff
@@ -59,8 +58,8 @@ func CheckEnvVars() error {
 			allEnvIsSet = false
 		}
 	}
-	if allEnvIsSet == false {
-		finalError := fmt.Sprintf("%s", errorString)
+	if !allEnvIsSet {
+		finalError := errorString
 		return errors.New(finalError)
```

Comment:
@rodrigo-brito wouldn't it look better with the return statement only?

`return errors.New(errorString)`

Reply:
I agree!